doc/source/guide/glossary.rst

```diff
@@ -0,0 +1,92 @@
+.. _glossary:
```

Comment:
I'm currently writing a short initial text on discretization, where I'm going to put the long explanation from the glossary, leaving there only the short summary of each term.